lang/c++/build.sh

```diff
@@ -58,7 +58,7 @@ function do_doc() {
 function do_dist() {
   rm -rf $BUILD_CPP/
   mkdir -p $BUILD_CPP
-  cp -r api AUTHORS build.sh CMakeLists.txt ChangeLog \
+  cp -r include/avro AUTHORS build.sh CMakeLists.txt ChangeLog \
```

Comment:
```suggestion
  cp -r include AUTHORS build.sh CMakeLists.txt ChangeLog \
```
I hope this is what is expected!
Now it will copy `include/` with all its subfolders.
With `cp -r include/avro ...` it will copy just `avro/` and put it in the root of $BUILD_CPP